src/helm-frontend/src/components/ScenariosList.tsx

```diff
@@ -33,22 +34,46 @@ export default function ScenariosList({ runGroups }: Props) {
           )
           .map((topLevelGroup, idx) => (
             <li key={idx}>
-              <h2>{topLevelGroup.display_name}</h2>
-              <ul>
+              <ReactRouterLink
+                className="text-black"
+                to={"groups/" + topLevelGroup.name}
+              >
+                <h2>{topLevelGroup.display_name}</h2>
+              </ReactRouterLink>
+              <ul className="list-disc list-inside">
                 {subGroups
                   .filter((subGroup) =>
                     (topLevelGroup.subgroups || []).includes(subGroup.name),
                   )
-                  .map((subGroup, idx) => (
-                    <li
-                      key={idx}
-                      className={`${
-                        subGroup.todo ? "ml-4 text-slate-300" : "ml-4"
-                      }`}
-                    >
-                      {subGroup.display_name}
-                    </li>
-                  ))}
+                  .map((subGroup, idx) =>
+                    subGroup.todo || subGroup.name.includes("CLEVA") ? (
```

Comment:
We should just set todo to true for all the CLEVA results and then remove this `.includes()` bit.